Makefile

```diff
@@ -132,4 +132,5 @@ install-moq:
 	go install github.com/matryer/moq@latest
 
 generate: install-moq
+	find . | grep _moq.go | xargs rm
```

Comment:
`moq` isn't smart enough to overwrite the current _moq.go files prior to running `go generate` so this manually clears them out.